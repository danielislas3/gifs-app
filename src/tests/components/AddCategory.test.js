import React from "react";
const { shallow } = require("enzyme");
const { AddCategory } = require("../../components/AddCategory");

describe("pruebas en <AddCategory/>", () => {
  const setCategories = () => {};
  test("debe de mostrarse correctamente", () => {
    const wrapper = shallow(<AddCategory setCategories={setCategories} />);
    expect(wrapper).toMatchSnapshot();
  });
});
