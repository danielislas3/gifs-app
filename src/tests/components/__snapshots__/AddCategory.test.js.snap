// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en <AddCategory/> debe de mostrarse correctamente 1`] = `
<form
  onSubmit={[Function]}
>
  <h2>
    Add Category
  </h2>
  <input
    name="cat"
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
